### ISSUE TITLE
Add support for requesting profile changes

### ISSUE THREAD

**Issue by mattst88** [mgorny/nattka#12]
Ruby stabilizations almost never receive `sanity-check+` because they require profiles changes to `RUBY_TARGETS`. See https://bugs.gentoo.org/688976 for example

If the package list field (which might need a rename :) could accept a syntax for profile changes we could do better checking of changes like this.

E.g., something like this, perhaps

```
use.stable.mask: -ruby_targets_ruby25
make.defaults: RUBY_TARGETS="ruby24 ruby25"

app-eselect/eselect-ruby-20190121
dev-ruby/did_you_mean-1.2.1
dev-ruby/rdoc-6.1.2
dev-lang/ruby-2.5.8
```

**Comment by mgorny** [comment]
I suppose it's start to thinking how to implement it. I suppose the first question is how fancy the support should be. Would it be sufficient to specify a file, and a change to be applied to it? Say, something like:

```
profiles/arch/hppa/use.mask: -gnome-keyring
```

that would append `-gnome-keyring` to the bottom? (which is kinda equivalent to removing `gnome-keyring` that is there)

**Comment by mattst88** [comment]
Yeah, I think that would work. I agree that there's a lot of ambiguity to just specifying `use.mask: -gnome-keyring` so specifying the whole path would be fine with me if that simplifies the problem.
